### ISSUE TITLE
Add other logging commands

### ISSUE THREAD

**Issue by andremw** [andremw/hubot-jira-logger#4]
I think it would be nice to have these commands as well:

**hubot start task <task id>**: starts counting time for the task
**hubot pause task <task id>**: pauses the time counter for the task
**hubot finish task <task id>**: logs work for the task and removes the counter
**hubot jira open tasks**: displays the tasks that are either started or paused
**hubot jira yesterday**: displays the tasks that were done in the previous work day (this makes it easier to check what I did)

This syntax is nice because you don't have to keep track of the spent time. You simply _start_ and _finish_ tasks and the bot will calculate it for you.

**Comment by faelperetta** [comment]
I think that you can split those commands in 2 enhancements. One for log hours commands (**start, pause finish**) and another one for report commands (**open tasks, last tasks**)

**Comment by RodrigoNascCarvalho** [comment]
I agree that we should split these into two enhancements. I feel the mechanism for starting, pausing and finishing tasks will be more complex than the ones for the reporting commands.

**Comment by andremw** [comment]
I agree. I created issue #7. Do you guys wanna help on this? :)

**Comment by RodrigoNascCarvalho** [comment]
I want to, but I'm busy right now :0

**Comment by faelperetta** [comment]
yep
